lib/activerecord-import/adapters/abstract_adapter.rb

```diff
@@ -16,7 +16,7 @@ def insert_many( sql, values, *args ) # :nodoc:
       sql2insert = base_sql + values.join( ',' ) + post_sql
       insert( sql2insert, *args )
 
-      number_of_inserts
+      [number_of_inserts,[]]
```

Comment:
I'm thinking of making this return an OpenStruct or some basic object in case we decide to ever include more information. That will allow us to do that without breaking the interface again. 

I originally obviously didn't think of that happening. :)